apps/src/dance/ai/DanceAiClient.ts

```diff
@@ -2,7 +2,7 @@ import UntypedCachedBackgroundEffects from '@cdo/static/dance/ai/model/cached_ba
 import UntypedCachedForegroundEffects from '@cdo/static/dance/ai/model/cached_foreground_effects_map.json';
 import UntypedCachedPalettes from '@cdo/static/dance/ai/model/cached_palettes_map.json';
 
-import {FieldKey, GeneratedEffect} from '../types';
+import {FieldKey, GeneratedEffect} from '@cdo/apps/dance/ai/types';
```

Comment:
Can this just be `'./types'`?

Reply:
It _can_, but I was concerned about it causing confusion whether it's the `dance/types` file or the `dance/ai/types` file.

Reply:
I did just rename the file to `ai-types` and used a local relative path import instead. Sound good?

Reply:
> Especially considering that the types file is currently in dance/ai, so these imports would just be ../types.

Wouldn't the import just be `./types` since `DanceAiClient` is in same directory? 
I don't have a strong preference but it seems like using `types` works for now - I usually just glance at the directory I'm currently in.

Reply:
I think using `@cdo/apps/dance/ai/types` when in `utils` would be appropriate.

Reply:
Yeah, I do have a slight preference for keeping the file named `types` - would it be clearer to keep the AI import as `./types` or `../types` and instead switch the generic dance types import to `@cdo/apps/dance/types` inside AI dance code?

Reply:
Okay, I know when I'm licked. 😆 

Moved `ai-types` back to `types`.

I think I'd rather just keep it as the `../types` import, though, for consistency's sake. Even though it's potentially slightly ambiguous, I think the consistent style is better, and making sure we're importing the right file isn't super onerous.